styles/_theme.scss

```diff
@@ -659,3 +397,28 @@ blockquote .source {
   padding: 0;
   list-style-position: inside;
 }
+
+.usa-prose {
+  .usa-collection__item {
+    border-top: none;
+  }
+}
+
+h1 {
+  @include u-font-family('sans')
+}
+
+.usa-intro, .usa-prose>h1+p {
+  max-width: 88ex;
+  @include u-font-family('sans');
+  font-size: 1.46rem;
```

Comment:
Ah, i just realized you're setting both font family and size. In that case, the right mixin to use is `u-font()` where you set the size and family together, because USWDS does some magic x-height normalization across fonts: https://designsystem.digital.gov/utilities/font-size-and-family/#utility-mixins-2

`@include u-font('body', 'lg')` (i'm not sure if the token is `lg` but it might be.